README.md

```diff
@@ -66,6 +66,7 @@ Mesos, designed for high performance data processing jobs that require flexibili
 - [Streamz](https://github.com/python-streamz/streamz) [Python] - A lightweight library for building pipelines to manage continuous streams of data; supports complex pipelines that involve branching, joining, flow control, feedback, back pressure, and so on.
 - [Stream Ops](https://github.com/nanosai/stream-ops-java) [Java] - A fully embeddable data streaming engine and stream processing API for Java.
 - [Tributary](https://github.com/timkpaine/tributary) [Python] - A python library for constructing dataflow graphs. Supports synchronous, reactive data streams built using python generators that mimic complex event processors, as well as lazily-evaluated acyclic graphs and functional currying streams.
+- [YoMo](https://github.com/yomorun/yomo) [Go] - Unix Pipeline with internet scale.
```

Comment:
The description doesn't show why the project is relevant here.

Reply:
hi @manuzhang , I considered this when adding, as I think `unix pipeline` is well defined, so I didn't explain more about the features. If it's not clear, I think I can add a link about it.